packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -62,14 +95,74 @@ export const CardHeader: React.FunctionComponent<CardHeaderProps> = ({
         </div>
       );
 
+      if (actions?.actions && !(isClickable && isSelectable)) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          `${
+            isClickable ? 'Clickable' : 'Selectable'
+          } only cards should not contain any other actions. If you wish to include additional actions, use a clickable and selectable card.`
+        );
+      }
+
+      const handleActionClick = (event: React.FormEvent<HTMLInputElement> | React.MouseEvent) => {
+        if (isClickable) {
+          if (selectableActions?.onClickAction) {
+            selectableActions.onClickAction(event);
+          } else if (selectableActions?.to) {
+            window.open(selectableActions.to, '_blank');
+          }
+        }
+      };
+
+      const getClickableSelectableProps = () => {
+        const baseProps = {
+          className: 'pf-m-standalone',
+          inputClassName: isClickable && !isSelectable && 'pf-v5-screen-reader',
+          label: <></>,
+          'aria-label': selectableActions?.selectableActionAriaLabel,
+          'aria-labelledby': selectableActions?.selectableActionAriaLabelledby,
+          id: selectableActions?.selectableActionId,
```

Comment:
`selectableActionId` prop is marked as required.  So either we do not need the check here or it was mistakenly marked required.